### DISCUSSION TITLE
Witgui Icon

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1344] [Icon request] [answered]
Hey, i would love an icon for this app: https://desairem.com/wordpress/witgui/

It's a Wii games backup manager for mac

Thanks,

**Comment by blaichch** [comment, marked as answer]
Here's my take on it:
<img width="512" alt="Witgui" src="https://user-images.githubusercontent.com/12624897/145987342-6095521d-2520-4a7e-abf4-b048a81f1608.png">

**Reply by ghost** [reply]
Thanks, it's perfect but I can use it as an icon as it's not the supported type of file, can you upload an .icns version please

**Reply by blaichch** [reply]
I can't upload it here on github. But I'll will upload it to macosicons.com. Until it is available you can download it [here](https://drive.google.com/file/d/1gOZrjADOLCA4N79-EUDMAc2cWatcXVds/view?usp=sharing).
